test/extended/machines/scale.go

```diff
@@ -44,7 +44,7 @@ func listWorkerMachineSets(dc dynamic.Interface) ([]objx.Map, error) {
 	machineSets := []objx.Map{}
 	for _, ms := range objects(objx.Map(obj.UnstructuredContent()).Get("items")) {
 		e2e.Logf("Labels %v", ms.Get("spec.selector.matchLabels"))
```

Comment:
Should this also be logging the labels from the template then?

Reply:
+1, done